runtime/near-vm-runner/src/cache.rs

```diff
@@ -17,6 +17,9 @@ pub(crate) fn compile_module(
     code: &[u8],
     config: &VMConfig,
 ) -> Result<wasmer_runtime::Module, VMError> {
+    let span = tracing::info_span!("compile_module");
+    let _span = span.enter();
```

Comment:
Can this be simplified to `tracing::info_span!("compile_module").enter();`

Reply:
hopefully soon: https://github.com/tokio-rs/tracing/pull/1252!